samples/dev-ocs-fio.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
```

Comment:
You pulled this file into your PR by accident.    This is part of the new fio automation